src/test/java/nl/tudelft/scrumbledore/BubbleTest.java

```diff
@@ -0,0 +1,41 @@
+package nl.tudelft.scrumbledore;
+
+//import static org.junit.Assert.*;
```

Comment:
I had in mind that this class will be tested more in the future. So I personally commented it out.